src/renderer/components/BalanceGraph/BalanceGraph.stories.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withKnobs, text, boolean, number, select } from '@storybook/addon-knobs';
+import { BalanceGraph } from './BalanceGraph';
+import { IntervalEnum } from './BalanceGraph';
+
+storiesOf('BalanceGraph', module).add('BalanceGraph', () => {
+    
+    return <BalanceGraph 
+    defaultInterval={IntervalEnum.DAY}
+    onOptionClick={()=>{console.log("")}}
+    getData={()=>{
+        return(
+            [2356,3213,8934,7924,7924,1245,3456,5673,2124,
```

Comment:
I just realized, since it's balance graph and balance is in gwei, those are gonna be huge numbers. It's better to use bn.js.
Can recharts render bn.js?